redis.conf

```diff
@@ -737,6 +737,15 @@ replica-priority 100
 # For more information about ACL configuration please refer to
 # the Redis web site at https://redis.io/topics/acl
 
+# ACL LOG
+#
+# The ACL Log tracks failed commands and authentication events associated
+# with ACLs. The ACL Log is useful to troubleshoot failed commands blocked 
+# by ACLs. The ACL Log is stored in and consumes memory. There is no limit
+# to its length.You can reclaim memory with ACL LOG RESET or set a maximum
```

Comment:
Missing space after the period.

Also "There is no limit to its length" kind of conflicts with the configuration directive's purpose.

Reply:
Was copying the slowlog comments :).

Will update shortly

Reply:
@itamarhaber  - fixes made in the last commit.